### PR TITLE
Add list_action_button_content and logo_content options

### LINKED ISSUE #6970: Option to get small `link_action` buttons.
## Feature Request

What do you think about an option to get the actions buttons with only the icons @sonata-project/contributors ?
Has it already been discussed ?

It would render something like
image

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

BC

I take in account the review from https://github.com/sonata-project/SonataAdminBundle/pull/7166

Closes  #6970

## Changelog

```markdown
### Added
- Option `list_action_button_content`.

### Deprecated
- Option `title_mode`. Use `logo_content` instead.
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Please review @sonata-project/contributors , then I'll do the 3.104 release :)

**Review by jordisala1991 — DISMISSED** [review]

File: src/Resources/views/standard_layout.html.twig [hunk 1]
```diff
@@ -125,10 +125,18 @@ file that was distributed with this source code.
                 {% block logo %}
                     {% apply spaceless %}
                         <a class="logo" href="{{ path('sonata_admin_dashboard') }}">
-                            {% if 'single_image' == sonata_config.getOption('title_mode') or 'both' == sonata_config.getOption('title_mode') %}
+                            {# NEXT_MAJOR: Remove the title_mode check #}
+                            {% if 'icon' == sonata_config.getOption('logo_content')
+                                or 'all' == sonata_config.getOption('logo_content')
+                                    and ('single_image' == sonata_config.getOption('title_mode') or 'both' == sonata_config.getOption('title_mode'))
```
Comment (phansys) [inline src/Resources/views/standard_layout.html.twig:131]:
```suggestion
                                and ('single_image' == sonata_config.getOption('title_mode') or 'both' == sonata_config.getOption('title_mode'))
```

File: src/Resources/views/CRUD/list__action_delete.html.twig [hunk 2]
```diff
@@ -10,12 +10,21 @@ file that was distributed with this source code.
...
     <a
         href="{{ admin.generateObjectUrl('delete', object, actions.link_parameters|default([])) }}"
         class="btn btn-sm btn-default delete_link"
         title="{{ 'action_delete'|trans({}, 'SonataAdminBundle') }}"
     >
-        <i class="fa fa-times" aria-hidden="true"></i>
+        {% if button_content == 'icon' or button_content == 'all' %}
+            <i class="fa fa-times" aria-hidden="true"></i>
+        {% endif %}
+        {% if button_content != 'text' and button_content != 'all' %}
```
Comment (phansys) [inline src/Resources/views/CRUD/list__action_delete.html.twig:22]: Maybe we could set the result of this condition in a variable to reuse the same logic for the closing `</span>`. We can also use a `if`/`else` condition.
Personally, I don't feel comfortable redefining the same logic twice.

**Comment by VincentLanglet** [comment]
I updated the logic @phansys, do you prefer this way ?

**Comment by phansys** [comment]
Thank you! It is consistent with what we have in other similar cases.

**Review by phansys — APPROVED** [review]

**Pull request merged**